src/index.ts

```diff
@@ -45,6 +50,10 @@ export default class extends Controller {
 
     await this.leave()
 
-    this.element.remove()
+    if (this.repeatValue) {
```

Comment:
I tested your PR and this condition causes a loop where the same notification keeps popping up.
To allow the notification to be shown more than once I just remove the `this.element.remove()` and everything works